extensions/notebook-renderers/src/index.ts

```diff
@@ -218,7 +218,8 @@ function findScrolledHeight(scrollableElement: HTMLElement): number | undefined
 }
 
 function scrollingEnabled(output: OutputItem, options: RenderOptions) {
-	return output.metadata?.scrollable !== undefined ? output.metadata?.scrollable : options.outputScrolling;
+	const metadata: any = output.metadata;
+	return metadata?.scrollable !== undefined ? metadata?.scrollable : options.outputScrolling;
```

Comment:
Unsure if casting to `any` is the most type-safe thing to do. We could instead do something like

```suggestion
	return typeof metadata === 'object' && metadata !== undefined && 'scrollable' in metadata && metadata?.scrollable !== undefined ? metadata?.scrollable : options.outputScrolling;
```

Alternatively perhaps the `metadata` property in the [notebook renderer](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/94cb5956432f48ef5ecc91163d584454db1f5073/types/vscode-notebook-renderer/index.d.ts#L49) types should be updated to be an indexable type `metadata: [key: string]: unknown`?